ui/components/src/TimeSeriesTooltip/TooltipContent.tsx

```diff
@@ -12,107 +12,51 @@
 // limitations under the License.
 
 import { useMemo } from 'react';
-import PinOutline from 'mdi-material-ui/PinOutline';
-import Pin from 'mdi-material-ui/Pin';
-import { Box, Divider, Stack, Typography } from '@mui/material';
-import { useTimeZone } from '../context/TimeZoneProvider';
+import { Box } from '@mui/material';
 import { NearbySeriesArray } from './nearby-series';
 import { SeriesInfo } from './SeriesInfo';
 
 export interface TooltipContentProps {
   series: NearbySeriesArray | null;
-  isTooltipPinned: boolean;
   wrapLabels?: boolean;
-  onUnpinClick: () => void;
 }
 
 export function TooltipContent(props: TooltipContentProps) {
-  const { series, wrapLabels, isTooltipPinned, onUnpinClick } = props;
-  const { formatWithUserTimeZone } = useTimeZone();
-
-  const seriesTime = series && series[0] && series[0].date ? series[0].date : null;
-
-  const formatTimeSeriesHeader = (timeMs: number) => {
-    const date = new Date(timeMs);
-    const formattedDate = formatWithUserTimeZone(date, 'MMM dd, yyyy - ');
-    const formattedTime = formatWithUserTimeZone(date, 'HH:mm:ss');
-    return (
-      <Box>
-        <Typography
-          variant="caption"
-          sx={(theme) => ({
-            color: theme.palette.common.white,
-          })}
-        >
-          {formattedDate}
-        </Typography>
-        <Typography variant="caption">
-          <strong>{formattedTime}</strong>
-        </Typography>
-      </Box>
-    );
-  };
```

Comment:
It's really nice how this is getting organized + split up into Header and Content, it's nice to do some divvying since there's a lot of logic 👏